model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,91 @@
+groups:
+  - id: metric.telemetry.sdk.span.ended_count
+    type: metric
+    metric_name: telemetry.sdk.span.ended_count
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.sampled_count
+    type: metric
+    metric_name: telemetry.sdk.span.sampled_count
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.processor.queue_size
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_size
+    stability: experimental
+    brief: "The number of spans in the queue of a given instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
```

Comment:
only batch processor has a queue, perhaps we can define a batch-processor specific metric instead?

Reply:
I was thinking of allowing custom processors to also use this metric, if they make use of queuing. But I don't have a strong opinion here.